keel-core/src/main/kotlin/com/netflix/spinnaker/keel/persistence/ResourceStateHistoryEntry.kt

```diff
@@ -0,0 +1,8 @@
+package com.netflix.spinnaker.keel.persistence
+
+import java.time.Instant
+
+data class ResourceStateHistoryEntry(
+  val state: ResourceState,
+  val timestamp: Instant
```

Comment:
I figure we can add more to this type. The details of the detected diff, for example.